packages/firestore/test/integration/util/helpers.ts

```diff
@@ -111,6 +107,23 @@ function apiDescribeInternal(
   }
 }
 
+type ApiSuiteFunction = (
+  message: string,
+  testSuite: (persistence: boolean) => void
+) => void;
+interface ApiDescribe {
+  (message: string, testSuite: (persistence: boolean) => void): void;
+  skip: ApiSuiteFunction;
+  only: ApiSuiteFunction;
+}
+
+export const apiDescribe = apiDescribeInternal.bind(
+  null,
+  describe
+) as ApiDescribe;
+apiDescribe.skip = apiDescribeInternal.bind(null, describe.skip);
+apiDescribe.only = apiDescribeInternal.bind(null, describe.only);
+
```

Comment:
Without these changes, we were getting two errors:

> Argument of type 'PendingSuiteFunction' is not assignable to parameter of type 'IContextDefinition'. Type 'PendingSuiteFunction' is missing the following properties from type 'IContextDefinition': only, skip

For passing `describe.skip` to `apiDescribeInternal()` and:

> Property 'skip' does not exist on type '(message: string, testSuite: (persistence: boolean) => void) => void'.ts(2339)

for assigning to `apiDescribe.skip`.